src/Magick.NET/Statistics/PerceptualHash.cs

```diff
@@ -75,10 +78,17 @@ public double SumSquaredDistance(IPerceptualHash other)
     {
         Throw.IfNull(nameof(other), other);
 
+        var red = other.GetChannel(PixelChannel.Red);
```

Comment:
Result of updating interfaces: it highlights that `GetChannel` method is used internally assuming R/G/B channels exists.

So I updated the method... Related tests are not yet updated because I would like your GO/NOGO on this.

Reply:
Like this?

```c#
if (red == null || green == null || blue == null)
{
    throw new NotSupportedException("other IPerceptualHash must have Red, Green and Blue channel");
}
```

Reply:
Yup

Reply:
Should be ok now; waiting pipeline :)